test/e2e/e2e_test.go

```diff
@@ -0,0 +1,187 @@
+//go:build e2e
```

Comment:
Let's rename this test to e2e_scale_test.go

Reply:
renamed to `node_scale_test`, as `scale_test` is misleading as it might mean scalability test.